internal/dataplane/parser/translate_httproute_test.go

```diff
@@ -1305,3 +1245,11 @@ func TestIngressRulesFromHTTPRoutes_RegexPrefix(t *testing.T) {
 func HTTPMethodPointer(method string) *gatewayv1beta1.HTTPMethod {
 	return (*gatewayv1beta1.HTTPMethod)(&method)
 }
+
+func commonRouteSpecMock() gatewayv1beta1.CommonRouteSpec {
```

Comment:
Maybe accept a name param?

Reply:
I changed that, but given it's a mock it makes no difference.

Reply:
Linter is not happy :D
```
Error: internal/dataplane/parser/translate_httproute_test.go:1160:26: `commonRouteSpecMock` - `parentReferentName` always receives `"fake-gateway"` (unparam)
func commonRouteSpecMock(parentReferentName string) gatewayv1beta1.CommonRouteSpec {
 ```